lib/eventFactory.js

```diff
@@ -667,7 +674,7 @@ class EventFactory extends BaseFactory {
                     if (prRef) {
                         modelConfig.pr.ref = prRef;
 
-                        return p.syncPR(prNum);
+                        return subscribedEvent ? p : p.syncPR(prNum);
```

Comment:
for subscribed event, does `modelConfig.pr` need to change ? What's the implication ?